src/unitxt/metrics.py

```diff
@@ -1769,8 +1769,11 @@ def compute(
         references: List[List[Any]],
         predictions: List[Any],
         task_data: List[Dict],
+        reference_extraction_func: Callable[
```

Comment:
This is not needed, we always that the first reference in NER (we assume there is only one reference).
In unitxt, FM eval a reference is a full "correct answer".    Note in this case an answer is a  List[Tuple[str,str,str]].